lib/streamlit/elements/json.py

```diff
@@ -70,6 +70,9 @@ def json(
 
         if not isinstance(body, str):
             try:
+                for key in body:
+                    if type(body[key]) == set:
```

Comment:
Can you add a little comment here explaining why the code exists, for future readers? Something like

```
# Convert sets into lists, which render more nicely on the frontend
```

Reply:
Sure, I've added comments both about this and about mutability 👍